var/spack/repos/builtin/packages/py-pyfr/package.py

```diff
@@ -52,7 +56,48 @@ class PyPyfr(PythonPackage):
 
     # Optional dependencies
     depends_on("py-scipy", when="+scipy", type=("build", "run"))
+    depends_on("metis@5.0:", when="+metis", type=("run"))
     depends_on("scotch@6.0:", when="+scotch", type=("run"))
-    depends_on("cuda@8.0:", when="+cuda", type=("run"))
-    depends_on("rocblas@4.5.0:", when="+hip", type=("run"))
-    depends_on("libxsmm@1.18:+shared blas=0 +large_jit_buffer", when="+libxsmm", type=("run"))
+    depends_on("cuda@8.0: +allow-unsupported-compilers", when="+cuda", type=("run"))
```

Comment:
I think we require CUDA 11.4 greater for the UUID-query functions.

Reply:
done